### PR TITLE
reformat changie.yaml to ignore dependabot

### LINKED ISSUE #5460: [CT-830] [Bug] Changie not ignoring dependabot in contributors list
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

`dependabot` is not ignored from the contributors list when we generate changelog markdown via `changie batch`

### Expected Behavior

Do not list `dependabot [bot]` as a contributor.

Changes required to [this line](https://github.com/dbt-labs/dbt-core/blob/c25260e5dd2afa237a30db115605ece9629443d1/.changie.yaml#L34) to be `dependabot[bot]`

### Steps To Reproduce

Run `changie batch` with a dependabot changelog entry.  The change should appear but dependabot should not be a contributor.

### PR DESCRIPTION
**Pull request by McKnight-42**
resolves #5460 

### Description

minor name change in `changie.yaml` to cover the ignore for `dependabot` by changing its name to `dependabot[bot]`

### Checklist

- [x] I have read [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md) and understand what's expected of me
- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [ ] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [x] I have [opened an issue to add/update docs](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose), or docs changes are not required/relevant for this PR
- [ ] I have run `changie new` to [create a changelog entry](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#Adding-CHANGELOG-Entry)

### CODE REVIEW

File: tests/functional/artifacts/data/state/v6/manifest.json [hunk 1]
```diff
@@ -1 +1 @@
...
... [], "depends_on": {"macros": []}, "description": "", "meta": {}, "docs": {"show": true}, "patch_path": null, "arguments": [], "created_at": 1657733348.618489}, "macro.dbt_postgres.postgres__listagg": {"unique_id": "macro.dbt_postgres.postgres__listagg", "package_name": "dbt_postgres", "root_path": "/Users/matthewmcknight/git/dbt-core/plugins/postgres/dbt/include/postgres", "path": "macros/utils/listagg.sql", "original_file_path": "macros/utils/listagg.sql", "name": "postgres__listagg", "macro_sql": "{% macro postgres__listagg(measure, delimiter_text, order_by_clause, limit_num) -%}\n\n    {% if limit_num -%}\n    array_to_string(\n        (array_agg(\n            {{ measure }}\n            {% if order_by_clause -%}\n            {{ order_by_clause }}\n            {%- endif %}\n        ))[1:{{ limit_num }}],\n        {{ delimiter_text }}\n        )\n    {%- else %}\n    string_agg(\n        {{ measure }},\n        {{ delimiter_text }}\n        {% if order_by_clause -%}\n        {{ order_by_clause }}\n        {%- endif %}\n        )\n    {%- endif %}\n\n{%- endmacro %}", "resource_type": "macro", "tags": [], "depends_on": {"macros": []}, "description": "", "meta": {}, "docs": {"show"
```
Comment (emmyoop) [inline tests/functional/artifacts/data/state/v6/manifest.json:1]: Why did this change?

**Comment by emmyoop** [comment]
You can add the `skip changelog` label to this.  This won't require a changelog entry.

File: tests/functional/artifacts/data/state/v6/manifest.json (hunk 1 above)
Comment (McKnight-42) [reply tests/functional/artifacts/data/state/v6/manifest.json:1]: I'm mot entirely sure...the only change i'm seeing doing quick search between it is it mentions my name isntead of jerco in diff? during push only said one file was changed so little confused here.

Comment (McKnight-42) [reply tests/functional/artifacts/data/state/v6/manifest.json:1]: looks like addition of grants macros in manifest is making it look different?

Comment (emmyoop) [reply tests/functional/artifacts/data/state/v6/manifest.json:1]: Please remove the file.  This is unrelated to the change we're making.  And these test files really should not be modified as it defeats the purpose of testing against our manifest.  If they need to be modified to get the schema tests to work we really should be publishing a new schema.

Comment (McKnight-42) [reply tests/functional/artifacts/data/state/v6/manifest.json]: done!

**Review by emmyoop — APPROVED** [review]
Looks good!

**Pull request merged**